### DISCUSSION TITLE
http_status is 200,but trace is error

### DISCUSSION THREAD

**Discussion by wangan0915** [apache/skywalking#11017] [Q&A] [answered]
i throw a new Custom exception that name is ChannelBizException in my project,
and handle it in my globalExceptinHandler,return http_status is 200,but i found trace is error in skywalking.
skywalking how to define success and error in trace
![image](https://github.com/apache/skywalking/assets/39958329/f847bf39-2c79-4ad7-a1e0-4734f94f03a1)
![image](https://github.com/apache/skywalking/assets/39958329/63dff267-855b-4ccc-b0f8-e003cfa84e73)

**Comment by wu-sheng** [comment, marked as answer]
Read doc first, then ask. We should not take the load to help you to find the doc link

https://skywalking.apache.org/docs/skywalking-java/v8.16.0/en/setup/service-agent/java-agent/how-to-tolerate-exceptions/
